src/lib/api/api.go

```diff
@@ -558,16 +581,18 @@ func (a *API) GetLabels(w http.ResponseWriter, r *http.Request) {
 	}
 	a.ResponseStatus = http.StatusText(200)
 	a.Data = uniqueLabels
-	a.send(w, a.marshalled())
+	a.send(w, a.marshalled(), a.Message)
 }
 
 func (a *API) marshalled() []byte {
 	response := struct {
-		Status string      `json:"status"`
-		Data   interface{} `json:"data"`
+		Status  string      `json:"status"`
+		Data    interface{} `json:"data"`
+		Message string      `json:"message"`
 	}{
-		Status: a.ResponseStatus,
-		Data:   a.Data,
+		Status:  a.ResponseStatus,
+		Data:    a.Data,
+		Message: a.Message,
```

Comment:
please change this func to

```golang
func marshalled(status string, data interface{}) []byte {
	response := struct {
		Status string      `json:"status"`
		Data   interface{} `json:"data"`
	}{
		Status: status,
		Data:   data,
	}
	js, err := json.Marshal(response)
	if err != nil {
		fmt.Fprintln(os.Stderr, "json-marshal error: ", err)
	}

	return js
}
```
If this is stuck to a.*, then race problems will occur, Hence, make this independent.